app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -582,6 +594,14 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
     }
   }
 
+  .checkbox-form-wrapper {
+    padding: 32px;
```

Comment:
since we're using Crayons spacing units above, can we do that here?

According to `_import.scs`s, `32px` can mean `$su-7`